hack/README.md

```diff
@@ -8,50 +8,31 @@ This README documents:
 
 This is the script to update clientset/informers/listers and API deepcopy code using [code-generator](https://github.com/kubernetes/code-generator).
 
-Make sure to run this script after making changes to /pkg/apis/volumesnapshot/v1beta1/types.go.
-
-To run this script, you have to patch it:
-```patch
-diff --git a/hack/update-generated-code.sh b/hack/update-generated-code.sh
---- a/hack/update-generated-code.sh
-+++ b/hack/update-generated-code.sh
-@@ -27,7 +27,7 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-ge
- #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
- #                  instead of the $GOPATH directly. For normal projects this can be dropped.
- bash ${CODEGEN_PKG}/generate-groups.sh "deepcopy,client,informer,lister" \
--  github.com/kubernetes-csi/external-snapshotter/pkg/client github.com/kubernetes-csi/external-snapshotter/pkg/apis \
-+  github.com/kubernetes-csi/external-snapshotter/v2/pkg/client github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis \
-   volumesnapshot:v1beta1 \
-   --go-header-file ${SCRIPT_ROOT}/hack/boilerplate.go.txt
-```
-
-Once you are done with patching, continue:
-```bash
-rm -fr v2/
-ln -sfvn $(pwd) v2
-rm -fr pkg/client
-```
+Make sure to run this script after making changes to /client/apis/volumesnapshot/v1beta1/types.go.
 
-Run: ./hack/update-generated-code.sh from the project root directory.
+To run this script, Run: ./hack/update-generated-code.sh from the project root directory.
 
-Do not forget to revert previously applied workaround:
+Once you run the script, you got the output as:
 ```bash
-git checkout -- v2/
-git checkout -- hack/update-generated-code.sh
+Generating deepcopy funcs
+Generating clientset for volumesnapshot:v1beta1 at github.com/kubernetes-csi/external-snapshotter/client/v2/clientset
+Generating listers for volumesnapshot:v1beta1 at github.com/kubernetes-csi/external-snapshotter/client/v2/listers
+Generating informers for volumesnapshot:v1beta1 at github.com/kubernetes-csi/external-snapshotter/client/v2/informers
+
 ```
 
+
 ## update-crd.sh
 
-This is the script to update CRD yaml files under ./config/crd/ based on types.go file.
+This is the script to update CRD yaml files under /client/config/crd/ based on types.go file.
 
-Make sure to run this script after making changes to /pkg/apis/volumesnapshot/v1beta1/types.go.
+Make sure to run this script after making changes to /client/apis/volumesnapshot/v1beta1/types.go.
 
 Follow these steps to update the CRD:
 
-* Run ./hack/update-crd.sh from root directory, new yaml files should have been created under ./config/crd/
+* Run ./hack/update-crd.sh from client directory, new yaml files should have been created under ./config/crd/
```

Comment:
This is not right.  If I'm under pkg/client, there's no ./hack there.  If I run ../hack/update-crd.sh, it will generate yaml under pkg/client/config.  You didn't check in the new config folder under pkg/client.  They are still under pkg/confg/crd.  Did you just copy them over?

Reply:
I think we should create a folder named "hack" under pkg/client and move update-crd.sh there.
Also update README with this info.

Reply:
@xing-yang yes, it should be "../hack/update-crd.sh" from client directory and config/crd are generated in client directory also. So if i update this to "../hack/update-crd.sh", then i think we don't need to create hack folder seperately.